src/ResourceManager/Media/Commands.Media.Test/ScenarioTests/MediaTests.ps1

```diff
@@ -0,0 +1,195 @@
+﻿# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+<#
+.SYNOPSIS
+Test Media related Cmdlets
+#>
+function Test-Media 
+{
+  $rgname = GetResourceGroupName
+  $location = "East US"
```

Comment:
Get location values by querying for th needed providers.  Using hard-coded location values prevents tests from being run in other environments (for example, test environments, national clouds)